controllers/useraccount/useraccount_controller.go

```diff
@@ -616,8 +664,9 @@ func newNSTemplateSet(userAcc *toolchainv1alpha1.UserAccount) *toolchainv1alpha1
 			Name:      userAcc.Name,
 			Namespace: userAcc.Namespace,
 		},
-		Spec: userAcc.Spec.NSTemplateSet,
+		Spec: *userAcc.Spec.NSTemplateSet,
 	}
+	setOwnerLabel(nsTmplSet, userAcc.Name)
```

Comment:
correct me if I'm wrong, but we don't need the label.

Reply:
right, it's an outdated change (before you suggested me to watch the NSTemplateSet by name, using the UserAccount name). Removed in 865c85f